N9TQ-BURLINGTON.md

```diff
@@ -0,0 +1,12 @@
+# BURLINGTON, WI
+
+### Restaurants
+- The Sly Fox
```

Comment:
Hey, the Sly Fox has been closed and is for sale.  
Too bad because they had a great :fish:  fry on Fridays!